lib/hours.php

```diff
@@ -665,646 +626,598 @@ function import_hours() {
 			$baseName = $name;
 	}
-	return $baseName;	
+	return $baseName;
 }
 
-function locationNameToId($name) {
-	$baseName = nameLookup($name);
+function locationNameToId( $name ) {
+	$baseName = nameLookup( $name );
 	$id = 0;
-	
+
 	$args = array(
 		'post_type' => 'hours',
 		'posts_per_page' => -1,
 		'search_prod_title' => $baseName,
 		'post_parent' => 0,
 	);
-	
+
 	add_filter( 'posts_where', 'title_filter', 10, 2 );
 	$locationList = new WP_Query( $args );
 	remove_filter( 'posts_where', 'title_filter', 10, 2 );
-	
-	if ($locationList->have_posts()) {
+
+	if ( $locationList->have_posts() ) {
 		$locationList -> the_post();
-		
+
 		return get_the_ID();
-		
+
 	};
-	
-	// didn't find
+
+	// Didn't find.
 	return 0;
-	
-	
-}
 
+}
 
-function locationNameToIdOriginal($name) {
-	// looks for a manual match to the import name (names are converted to lowercase to eliminate capitalizations changes)
 
-	$baseName = nameLookup($name);
+function locationNameToIdOriginal( $name ) {
+	// Looks for a manual match to the import name (names are converted to lowercase to eliminate capitalizations changes).
+	$baseName = nameLookup( $name );
 	$id = 0;
-	
+
 	$args = array(
 		'post_type' => 'hours',
 		'posts_per_page' => -1,
 		'search_prod_title' => $baseName,
 		'post_parent' => 0,
 	);
-	
+
 	add_filter( 'posts_where', 'title_filter', 10, 2 );
 	$locationList = new WP_Query( $args );
 	remove_filter( 'posts_where', 'title_filter', 10, 2 );
-	
-	if ($locationList->have_posts()) {
+
+	if ( $locationList->have_posts() ) {
 		$locationList -> the_post();
-		$id = get_post_meta(get_the_ID(), "associated_location", true);
-		
+		$id = get_post_meta( get_the_ID(), 'associated_location', true );
+
 		return $id;
-		
+
 	};
-	
-	// didn't find
+
+	// Didn't find.
 	return 0;
-	
-	
+
 }
 
 
-function title_filter( $where, &$wp_query )
-{
+function title_filter( $where, &$wp_query ) {
 	global $wpdb;
 	if ( $search_term = $wp_query->get( 'search_prod_title' ) ) {
 		$where .= ' AND ' . $wpdb->posts . '.post_title LIKE \'%' . esc_sql( like_escape( $search_term ) ) . '%\'';
 	}
 	return $where;
 }
 
-function process_overview($arSheet) {
+function process_overview( $arSheet ) {
 	$startRow = 4;
-	
+
 	$rObj = array();
-	
-	foreach($arSheet as $row => $item) {
-		if ($row >= $startRow ) {
-			if ($item[A] != "") {
-				$semester = $item[A];
-				$start = strtotime($item[B]);
-				$end = strtotime($item[C]);
-				
+
+	foreach ( $arSheet as $row => $item ) {
+		if ( $row >= $startRow ) {
+			if ( $item[ A ] != '' ) {
+				$semester = $item[ A ];
+				$start = strtotime( $item[ B ] );
+				$end = strtotime( $item[ C ] );
+
 				$temp = array();
-				
-				echo "<b>$semester</b>: ".date("Y-m-d", $start)." to ".date("Y-m-d", $end)." <br>";
-				
-				$temp[semester] = $semester;
-				$temp[start] = $start;
-				$temp[end] = $end;
-				
-				array_push($rObj, $temp);
+
+				echo "<b>$semester</b>: " . date( 'Y-m-d', $start ) . ' to ' . date( 'Y-m-d', $end ) . ' <br>';
```

Comment:
My guess is that there's something about the $semester variable that requires double quotes rather than single quotes. I believe PHP parses one's file contents for variables (which appears to be happening in this case).